app/lib/pdf_filler/md502_pdf.rb

```diff
@@ -16,36 +16,48 @@ def initialize(submission)
-        "Enter Y of income more than $11,000": @xml_document.at("Form502 Income InvestmentIncomeIndicator")&.text == "X" ? "Y" : "",
-        "Enter day and month of Fiscal Year beginning": formatted_date(@xml_document.at('ReturnHeaderState TaxPeriodBeginDt')&.text, "%m-%d"),
-        "Enter day and month of Fiscal Year Ending": formatted_date(@xml_document.at('ReturnHeaderState TaxPeriodEndDt')&.text, "%m-%d"),
-        "Enter social security number": @xml_document.at('Primary TaxpayerSSN')&.text,
+        'Enter 1': @xml_document.at("Form502 Income FederalAdjustedGrossIncome")&.text,
+        'Enter 1a': @xml_document.at("Form502 Income WagesSalariesAndTips")&.text,
+        'Enter 1b': @xml_document.at("Form502 Income EarnedIncome")&.text,
+        'Enter 1dEnter 1d': @xml_document.at("Form502 Income TaxablePensionsIRAsAnnuities")&.text,
+        'Enter Y of income more than $11,000': @xml_document.at("Form502 Income InvestmentIncomeIndicator")&.text == "X" ? "Y" : "",
+        'Enter day and month of Fiscal Year beginning': formatted_date(@xml_document.at('ReturnHeaderState TaxPeriodBeginDt')&.text, "%m-%d"),
+        'Enter day and month of Fiscal Year Ending': formatted_date(@xml_document.at('ReturnHeaderState TaxPeriodEndDt')&.text, "%m-%d"),
+        'Enter social security number': @xml_document.at('Primary TaxpayerSSN')&.text,
         'Enter spouse\'s social security number': @xml_document.at('Secondary TaxpayerSSN')&.text,
-        "Enter your first name": @xml_document.at('Primary TaxpayerName FirstName')&.text,
-        "Enter your middle initial": @xml_document.at('Primary TaxpayerName MiddleInitial')&.text,
-        "Enter your last name": @xml_document.at('Primary TaxpayerName LastName')&.text,
+        'Enter your first name': @xml_document.at('Primary TaxpayerName FirstName')&.text,
+        'Enter your middle initial': @xml_document.at('Primary TaxpayerName MiddleInitial')&.text,
+        'Enter your last name': @xml_document.at('Primary TaxpayerName LastName')&.text,
         'Enter Spouse\'s First Name': @xml_document.at('Secondary TaxpayerName FirstName')&.text,
         'Enter Spouse\'s middle initial': @xml_document.at('Secondary TaxpayerName MiddleInitial')&.text,
         'Enter Spouse\'s last name': @xml_document.at('Secondary TaxpayerName LastName')&.text,
         'Enter 4 Digit Political Subdivision Code (See Instruction 6)': @xml_document.at('MarylandSubdivisionCode')&.text,
         'Enter Maryland Political Subdivision (See Instruction 6)': @submission.data_source.political_subdivision,
         'Enter zip code + 5': @submission.data_source.residence_county,
-        "Check Box - 1": filing_status(:filing_status_single?) ? 'Yes' : 'Off',
-        "Check Box - 2": filing_status(:filing_status_mfj?) ? 'Yes' : 'Off',
-        "Check Box - 3": filing_status(:filing_status_mfs?) ? 'No' : 'Off', # "No" is the checked option
+        'Check Box - 1': filing_status(:filing_status_single?) ? 'Yes' : 'Off',
```

Comment:
lint was complaining about the double quotes so changed them all to single quotes